src/main/rx/rx.c

```diff
@@ -374,10 +398,10 @@ bool rxUpdateCheck(timeUs_t currentTimeUs, timeDelta_t currentDeltaTime)
 
             if (frameStatus & (RX_FRAME_FAILSAFE | RX_FRAME_DROPPED)) {
             	// No (0%) signal
-            	setRssi(0, RSSI_SOURCE_FRAME_ERRORS);
+            	setFrameQuality(false);
```

Comment:
When `RSSI_SOURCE_FRAME_ERRORS` was introduced, it was decided to have the frame error based link quality represented as RSSI, to make it available in a seamless way.
I think we should stick to this. Also, if this is done, the averaging is done by the RSSI averaging, avoiding code duplication.

Reply:
My intention is not to change the existing functionality, but to additionally make the frame error rate available separately. The code duplication I can tidy-up. Though thinking about it, frame errors is effectively a count so maybe something a bit more Poisson is appropriate.

I tried this today and I could clearly see I had > 80%  to about 48-50 rssi which improved my confidence when flying at that part of the field.

Reply:
Fair enough, let me know when the de-duplication is done.